### ISSUE TITLE
Add a page to display upcoming contests for codeforces ; file name : codeforces.html

### ISSUE THREAD

**Issue by Rohxn16** [Rohxn16/CPScrape#4]
The page is going to inherit base.html like the rest of the pages with the only difference being in the content part where it will deviate from the usual card style display and display upcoming contests in the form of a list (ofc the list needs to be dynamic so what can be done is to add a static dummy list there which I will later modify to be dynamic). **Assignee is requested to maintain the folder structure and refrain from making any changes other than the html file that the assignee is working on.**

**Comment by parthasdey2304** [comment]
Got it! Assign that to me, let me try to make it dynamic first using some APIs of CodeForces (if I find any). If its not possible I would try using a dummy list.

**Comment by parthasdey2304** [comment]
https://codeforces.com/apiHelp/objects#Contest
I guess I have found one API, let me see if I can implement this here...

**Comment by Rohxn16** [comment]
> https://codeforces.com/apiHelp/objects#Contest I guess I have found one API, let me see if I can implement this here...

No, that is not what I am talking abt, I need you to create a simple webpage that inherits the base html file and and displays a dummy list with 2 or 3 list items in it. The style needs to be glassmorphic thats all that is needed here.
